src/patterns/start-pages/index.md.njk

```diff
@@ -1,41 +1,73 @@
+
+The content team at GDS is responsible for creating mainstream start points on [GOV.UK](http://gov.uk/), as with other mainstream content. They’ll work with you to make sure the journey from [GOV.UK](http://gov.uk/) into the service works as well as possible. It’s a good idea to [talk to your organisation’s content team during alpha](https://www.gov.uk/service-manual/service-assessments/get-your-service-on-govuk#before-your-alpha-assessment) - if necessary, they can put you in touch with the team at GDS.
+
+
+### Mainstream start points
+
+For a mainstream start point, the options are:
+- a simple start page
+- a start point within a multipart guide
+
+Use a simple start page if users can start using the service without needing much context.
+
+<img src="simple-start-page.png" alt="A screenshot showing an example of a simple start page from the coronavirus shielding support service. The page includes a heading, text to explain who can use the service, and a start button labelled 'Register or update your details'.">
+
+If the start point is linked to a wider process that needs more explanation, create an ‘Apply’ section within a multipart guide. 
 
-Start pages include 4 main elements:
+For example, before a user applies for probate (the right to manage someone’s estate after they die) they need to understand whether probate is required, whether they’re the right person to apply, how the details of the will and the tax situation affect what they’re supposed to do, how to get the relevant information together and how the different outcomes affect what they'll need to do next.
 
-1. The service name: this should help people understand what your service does and whether they need to use it - [learn more about naming your service](https://www.gov.uk/service-manual/design/naming-your-service).
+It’s important that users have this context, so it makes sense to embed the start point within the multipart guide — rather than creating the start point as a separate start page, allowing users to miss the context and potentially do the wrong thing.
 
-2. A short introduction to list things that most users will need to know, for example, what your service will let them do or how much it’ll cost.
+<img src="multipart-guide-start-point.png" alt="A screenshot showing an example of a start point within a multipart guide. The page is shown as page 4 of 5 and its body text explains what the user will need to start using the service, followed by a start button labelled 'Apply now'.">
 
-3. The ‘Start now’ call-to-action button.
+The Carer’s Allowance team found that, for their service, [providing a start point within a multipart guide helped users get to the right information without feeling overwhelmed](https://dwpdigital.blog.gov.uk/2014/07/24/improving-to-the-carers-allowance-guide/).
 
-4. A list of other ways to access your service.
+## Creating a start point using Whitehall Publisher
```

Comment:
Nests the Markdown code so it's understood to be part of the the GOV.UK publishing process. Also deemphasises it slightly given that it's not being removed.

```suggestion
### Creating a start point using Whitehall Publisher
```